estate/models/test_model.py

```diff
@@ -0,0 +1,124 @@
+from odoo import models, fields
+from datetime import datetime, timedelta
```

Comment:
```suggestion
from datetime import datetime, timedelta

from odoo import fields, models
```
Generally, we make external imports first and then imports from odoo in alphabetical order
https://www.odoo.com/documentation/18.0/contributing/development/coding_guidelines.html